deployment/assets/create-script.py

```diff
@@ -1,8 +1,8 @@
 import os
 import json
 
-MODELS_LIST_VAR = json.dumps(os.getenv('MODELS_LIST')) if os.getenv('MODELS_LIST') else  '["mini/artifacts"]'
-MODELS_LIST = json.loads(MODELS_LIST_VAR)
+# Variable must contain the path from models dir to where the model's artifacts are located
+MODELS_LIST = json.loads(os.getenv('MODELS_LIST', ["mini"]))
```

Comment:
json.loads wants a string, but the default ["mini"] has regressed back to a list.  Needs to be a string or maybe there shouldn't be a default anyway (w/ handling of None)